dispatcher/cli/dcos_spark/cli.py

```diff
@@ -8,6 +8,8 @@
     dcos spark run --help
     dcos spark run --submit-args=<spark-args>
                    [--dcos-space=<dcos_space>]
+                   [--kerberos-principal=kerberos_principal]
+                   [--keytab-secret-path=<keytab_secret>]
```

Comment:
Fixed. I also removed the `--tgt` section of the docs, because that doesn't actually exist AFAICT?